app/views/checkvariables.php

```diff
@@ -35,11 +35,12 @@
 $target = array_map(['Transvision\AnalyseStrings', 'cleanUpEntities'], $target);
 
 $mismatch = AnalyseStrings::differences($source, $target, $repo);
-
+$mismatch_number = count($mismatch);
 $bugzilla_link = 'https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&component='
                . Bugzilla::getURLencodedBugzillaLocale($locale, 'products')
                . '&product=Mozilla%20Localizations&status_whiteboard=%5Btransvision-feedback%5D';
 
+echo "<h2>" . $mismatch_number . " results found</h2>";
```

Comment:
4 comments:
1/ echo is fine but we tend to use "print" in out codebase
2/ don't use double quotes if you are not going to extrapolate variables
3/ since you use the mismatch_number only once, you can save a variable and out the count directly in the concatenation:
```php
print '<h2>' . count($mismatch) . ' results found</h2>';
```
4/ nitpick, what happens if you only have one result ? Your string would have a syntax error (1 results found) ;)